examples/simple_generic_component/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
You can avoid the whole `Cargo.toml` boilerplate by placing the `main.rs` file into `examples` like the other example files: https://doc.rust-lang.org/cargo/guide/project-layout.html